.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "standard",
```

Comment:
We've been using plain ol' Standard on Atom itself and some internal packages. Any particular reason we need to extend it for this?